packages/loader/src/KTXLoader.ts

```diff
@@ -23,7 +23,7 @@ export class KTXLoader extends Loader<Texture2D> {
           const parsedData = parseSingleKTX(bin);
           const { width, height, mipmaps, engineFormat } = parsedData;
           const mipmap = mipmaps.length > 1;
-          const texture = new Texture2D(resourceManager.engine, width, height, engineFormat, mipmap);
+          const texture = new Texture2D(resourceManager.engine, width, height, engineFormat, mipmap, undefined, false);
```

Comment:
Shouldn't always false

Reply:
we don't support sRGB with KTX1 now, and maybe we should delete ktx1 loader support